docs(ngMessageExp): Better description

spaceribs opened the pull request:
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

docs update

**What is the current behavior? (You can also link to an open issue here)**

ngMessageExp has the same description as ngMessage

**What is the new behavior (if this is a feature change)?**

Updated description

**Does this PR introduce a breaking change?**

No

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Other information**:


Old description was copied over from ngMessage.

spaceribs committed:
docs(ngMessageExp): Better description

Old description was copied over from ngMessage.

gkalpak commented on src/ngMessages/messages.js:
```diff
@@ -629,10 +629,8 @@ angular.module('ngMessages', [], function initAngularHelpers() {
-   * For `ngMessageExp` to operate, a parent `ngMessages` directive on a parent DOM element
-   * must be situated since it determines which messages are visible based on the state
-   * of the provided key/value map that `ngMessages` listens on.
+   * `ngMessageExp` is the same as {@link module:ngMessage `ngMessage`} but instead
```
Shouldn't this be `directive:ngMessage` or something (instead of `module:ngMessage`)?

gkalpak commented on src/ngMessages/messages.js:
```diff
@@ -629,10 +629,8 @@ angular.module('ngMessages', [], function initAngularHelpers() {
-   * must be situated since it determines which messages are visible based on the state
-   * of the provided key/value map that `ngMessages` listens on.
+   * `ngMessageExp` is the same as {@link module:ngMessage `ngMessage`} but instead
+   * of a DOM attribute, accepts an expression to be evaluated for the message key.
```
"DOM attribute" isn't correct. How about "static value"?

spaceribs committed:
Updating changes per feedback

link needs to point to the ngMessage directive, and DOM attribute isn't clear enough

spaceribs commented:
@gkalpak updated based on feedback, thanks
